modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase

```diff
@@ -0,0 +1,92 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-182586 - Verify it's possible to define output fields for my validation as partial validation"
+	@priority = 5
+	test CanAddEntry {
+		property custom.properties = "feature.flag.LPS-187846=true";
+		property portal.acceptance = "true";
+
+		task ("Given an custom object definition and a precision decimal field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 192638",
+				objectName = "CustomObject192638",
+				pluralLabelName = "Custom Objects 192638");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "PrecisionDecimal",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "BigDecimal",
+				isRequired = "false",
+				objectName = "CustomObject192638");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject192638");
+		}
+
+		task ("and Given an expression builder to validate if does not contain some numbers and set to partial validation with the precision decimal") {
```

Comment:
and Given an expression builder validation set to partial validation with the precision decimal field